Stac2video improvements

yellowcap opened the pull request:
A bunch of improvements, moving towards closing the python side on this project for now.

- Pip installable
- Made sure docs are readable and up to date
- Added a few more controls to stac2video script
- Added more metadata to the video.geojson @nerik including a `"dates"` field containing a list of dates that went into each frame at the full level (different dates for different tiles, but merged dates into a list per frame for all videos together)

yellowcap committed:
allow more control over output, use stac catalog api v1

batpad commented on README.md:
````diff
@@ -36,17 +36,41 @@ To run, use your favourite method to serve a local web-server in this folder. Wi
 
+
+```
+stac2video --dst=/path/to/exiting/folder/videos --coordx=-9.15032 --coordy=38.72595 --start=2023-01-01 --end=2023-05-01 --zoom=12 --width=5 --height=3 --interval=1M
````
minor typo - s/exiting/existing

yellowcap committed:
stac2video improvements
- Pip installable CLI
- More CLI options
- Better metadata in videos.geojson, closes #8
